h2o-py/h2o/fusion.py

```diff
@@ -0,0 +1,122 @@
+# -*- encoding: utf-8 -*-
+"""
+Rapids expression fusion.
+
+:copyright: (c) 2016 H2O.ai
+:license:   Apache License Version 2.0 (see LICENSE for details)
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import h2o.expr
+
+class Fusion(object):
```

Comment:
Perhaps we can call it RapidOptimizer or AST Optimizer (however the latter is already used in Python world)

Reply:
`ExprOptimizer` ?

Reply:
Yes, that sounds much better!

Reply:
👍